src/sentry/runner/commands/cleanup.py

```diff
@@ -63,17 +66,28 @@ def debug_output(msg: str) -> None:
 
 def multiprocess_worker(task_queue: _WorkQueue) -> None:
     # Configure within each Process
-    import logging
+    from sentry.runner import configure
 
-    from sentry.utils.imports import import_string
+    configure()
 
-    logger = logging.getLogger("sentry.cleanup")
+    while True:
+        j = task_queue.get()
+        if j == _STOP_WORKER:
+            task_queue.task_done()
+            return
 
-    from sentry.runner import configure
+        model_name, chunk = j
+        try:
+            process_deletion_task(model_name, chunk)
+        except Exception:
+            logger.exception("Error in process_deletion_task")
+        finally:
+            task_queue.task_done()
 
-    configure()
 
+def process_deletion_task(model_name: str, chunk: list[int]) -> None:
```

Comment:
This method is called exactly the same way when we use multi-processing:
<img width="370" height="127" alt="Image" src="https://github.com/user-attachments/assets/21e267bd-c23d-4436-b6f4-8ebb0b12929d" />
and when we don't:
<img width="256" height="78" alt="Image" src="https://github.com/user-attachments/assets/1a64d455-2ff1-422b-9f75-d6df26b30ae2" />